### ISSUE TITLE
cannot find requestIfNeeded 

### ISSUE THREAD

**Issue by krtliv** [ivanvorobei/SPPermission#137]
I try to show SPPermission dialog only when there are permissions that still needed. but i cant find this method.

`Type 'SPPermission.Dialog' has no member 'requestIfNeeded'`

Here is my PodFile

```
pod 'SPPermission/Notification'
pod 'SPPermission/Camera'
pod 'SPPermission/Location'
pod 'SPPermission/Microphone'
pod 'SPPermission/PhotoLibrary'
pod 'SPPermission/MediaLibrary'
```

**Comment by mido89** [comment]
i have same issue

**Comment by ivanvorobei** [comment]
Which SPPermission version you use?

**Comment by ivanvorobei** [comment]
I am upload new version today with fix it

**Comment by mido89** [comment]
> I am upload new version today with fix it

i tried  pod update and  i didn't get any updates   ,   i think the repo not updated  !!!!

**Comment by ivanvorobei** [comment]
@mido89 I later upload new version, please, wait and don't panic.

**Comment by ivanvorobei** [comment]
Fixed in 4.1
